### PR TITLE
chore: Additions to styleguide

### PR DESCRIPTION
**Pull request by LekoArts**
Hi!

This adds "How to abbreviate a term" and some instructions about "Jargon" to the styleguide.
I've used https://developers.google.com/style/abbreviations and https://developers.google.com/style/jargon to help me with that (in case we want to expand this since my addition is the minimal one)

### CODE REVIEW

**Comment by rachelnabors** [comment]
I'm such a fan of the Google devs styleguide. These will be great additions!

File: STYLEGUIDE.md [hunk 1]
```diff
@@ -142,14 +154,20 @@ Some learners may not have heard this expression before. It may be difficult for
 
 Avoid using language that assumes someone's level of proficiency. Something that is difficult for someone new to programming may not be difficult for a senior engineer. This language can inadvertently alienate or insult a learner. Avoid words like "just," "easy," "simple," "senior," "hard."
 
+Use as little [jargon](https://dictionary.cambridge.org/dictionary/english/jargon) as necessary. Describe jargon in parentheses on first reference or link to a trusted definition.
+
 #### Do
 
 > You can authenticate your app with Clerk in three steps. Install Clerk with `npm install @clerk/nextjs`, add your environment keys, and then wrap your app in `<ClerkProvider />`, and add [control components](https://clerk.com/docs/components/overview). Visit our [Quickstarts](https://clerk.com/docs/quickstarts/overview) for a step-by-step guide written for your framework.
 
+> Clerk works great with progressive web apps (an app that's built using web platform technologies, but that provides a user experience like that of a platform-specific app) as it supports offline mode. You can use the [dynamic import pattern](https://www.patterns.dev/vanilla/dynamic-import) to improve loading speeds.
```
Comment (rachelnabors) [inline STYLEGUIDE.md:163]:
```suggestion
> Clerk supports offline mode, a feature that lets users use an app without being connected to data or wifi. This means Clerk works great with progressive web apps, apps built with web platform technologies that provide similar experiences to platform-specific apps. You can use the [dynamic import pattern](https://www.patterns.dev/vanilla/dynamic-import) to improve loading speeds.
```

**Review by rachelnabors — CHANGES_REQUESTED** [review]
Love it, a few suggestions!

File: STYLEGUIDE.md [hunk 2]
```diff
@@ -38,6 +38,18 @@ Tech has created many new words by associating two words with each other like "f
...
+
+If you want to abbreviate a term in your article, write it out fully first, then put the abbreviation in parentheses. If you want to make an abbreviation plural treat them as regular words, e.g. APIs, IDEs or OSes.
+
+#### Do
+
+> An abstract syntax tree (AST) is a tree representation of code. ASTs are a fundamental part of the way a compiler works.
+
+#### Don't 
+
+> An AST is a tree representation of code. ASTs are a fundamental part of the way a compiler works.
```
[the remark this answers is not in the document]
Comment (rachelnabors) [reply STYLEGUIDE.md:51]: This is in the "don't" section!

[the remark this answers is not in the document]
Comment (LekoArts) [reply STYLEGUIDE.md]: oh yeah, lol

**Review by rachelnabors — APPROVED** [review]

**Pull request merged**